internal/provider/validators.go

```diff
@@ -178,6 +178,40 @@ func validateConsulClusterSize(v interface{}, path cty.Path) diag.Diagnostics {
 	return diagnostics
 }
 
+func validateConsulClusterCIDR(v interface{}, path cty.Path) diag.Diagnostics {
```

Comment:
Validation functions for the IP Address and description.

I went with `net.ParseCIDR`. I know you mentioned another function `netmask` @loshz , but that one was only for IPV4. This seemed simple enough

Reply:
done 👍🏽